frontend/src/metabase/lib/formatting.js

```diff
@@ -842,7 +842,7 @@ export function stripId(name: string) {
 }
 
 export function slugify(name: string) {
-  return name && name.toLowerCase().replace(/[^a-z\u0400-\u04ff0-9_]/g, "_");
+  return name && encodeURIComponent(name.toLowerCase());
```

Comment:
What about also replacing spaces with underscores?
`encodeURIComponent(name.toLowerCase().replace(/\s/g, '_'))`

I think that'd slightly beautify a common case of slugifying without risking non-ascii issues.